internal/controller/buildpipeline/buildpipeline_adapter.go

```diff
@@ -330,6 +330,12 @@ func (a *Adapter) handleUnsuccessfulPipelineRun(canRemoveFinalizer *bool) {
 	if h.HasPipelineRunFinished(a.pipelineRun) || a.pipelineRun.GetDeletionTimestamp() != nil {
 		// The pipeline run has failed
 		// OR pipeline has been deleted but it's still in running state (tekton bug/feature?)
+
+		// Add a failure annotation.  Used to propagate failure information to other PRs in group
+		FailureAnnotation = fmt.Sprintf("%s/%s", tekton.ResourceLabelSuffix, "failure")
```

Comment:
This should do the trick of resolving the check failures:
```suggestion
		FailureAnnotation := fmt.Sprintf("%s/%s", tekton.ResourceLabelSuffix, "failure")
```